README.md

```diff
@@ -1,158 +1,552 @@
+
+*equivalent CLI option is: `-docker-options`*
+
+##### Outer working directory
+
+```toml
+DOJO_WORK_OUTER="/tmp/my-project"
+```
+Defines directory on the docker host to mount into the container at `/dojo/work` (by default).
+The default is to mount current working directory.
+
+`Dojo` setups following mount `DOJO_WORK_OUTER:DOJO_WORK_INNER`.
+
+*equivalent CLI option is: `-work-dir-outer`*
+
+We do not recommend changing this.
+
+##### Inner working directory
+
+```toml
+DOJO_WORK_INNER="/dojo/work/my-project"
+```
+Defines directory inside the docker container, which is mounted from the host. By default `/dojo/work`.
+
+`Dojo` setups following mount `DOJO_WORK_OUTER:DOJO_WORK_INNER`.
+
+*equivalent CLI option is: `-work-dir-inner`*
+
+We do not recommend changing this.
+
+##### Identity directory
+
+```toml
+DOJO_IDENTITY_OUTER="/home/joe"
 ```
-$ dojo --image=alpine:3.8 whoami
-2019/01/25 13:50:37 [ 1]  INFO: (main.main) Dojo version 0.1.0
-2019/01/25 13:50:37 [ 7]  INFO: (main.DockerDriver.HandleRun) docker command will be:
- docker run --rm -v /home/ewa/code/dojo:/dojo/work -v /home/ewa:/dojo/identity:ro \
-   --env-file=/tmp/dojo-environment-dojo-dojo-2019-01-25_13-50-37-28841830 \
-   -ti --name=dojo-dojo-2019-01-25_13-50-37-28841830 alpine:3.8 whoami
-root
+Defines directory on the docker host to mount into the container at `/dojo/identity`.
+The default is to mount current user's home. See more about [identity directory](#home-and-identity-directory).
+
+`Dojo` setups following mount `DOJO_IDENTITY_OUTER:/dojo/identity`.
+
+*equivalent CLI option is: `-identity-dir-outer`*
+
+##### Blacklist variables
+
+```toml
+DOJO_BLACKLIST_VARIABLES="HOME,USER,MY_PREFIX_*"
 ```
+By default dojo will pass environment variables from host to the container.
+`DOJO_BLACKLIST_VARIABLES` can be used to control which variables should not be mapped.
+It is a list of environment variables, split by commas, which should not be transfered from host to the docker container.
 
-Run one command, save output to a variable:
+The default value is:
 ```
-$ my_stdout=$(dojo whoami)
-2019/01/25 13:51:39 [ 1]  INFO: (main.main) Dojo version 0.1.0
-2019/01/25 13:51:39 [17]  INFO: (main.DockerDriver.HandleRun) docker command will be:
- docker run --rm -v /home/ewa/code/dojo:/dojo/work -v /home/ewa:/dojo/identity:ro \
-  --env-file=/tmp/dojo-environment-dojo-dojo-2019-01-25_13-51-39-19563225 \
-  --name=dojo-dojo-2019-01-25_13-51-39-19563225 "alpine:3.8" whoami
-$ echo $my_stdout
-root
+"BASH*,HOME,USERNAME,USER,LOGNAME,PATH,TERM,SHELL,MAIL,SUDO_*,WINDOWID,SSH_*,SESSION_*,GEM_HOME,GEM_PATH,GEM_ROOT,HOSTNAME,HOSTTYPE,IFS,PPID,PWD,OLDPWD,LC*"
 ```
 
-Run one command, using configuration file, exit status is non 0.
+*equivalent CLI option is: `-blacklist`*
+
+##### Log level
+
+```toml
+DOJO_LOG_LEVEL="info"
 ```
-$ cat Dojofile
-DOJO_DOCKER_IMAGE="alpine:3.8"
+
+*In CLI use --debug=true or --debug=false*
```

Comment:
Could we use the double hyphen for each CLI option? `--debug`, `--blacklist`, etc. Golang binaries support single and double hyphens, but double hyphens are more unix standard.